version.txt

```diff
@@ -0,0 +1,2 @@
+v1.10.0-dev
+2021-01-13 18:02:20Z
```

Comment:
It seems space is not permitted as date-time separator:
- https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations
- https://stackoverflow.com/a/9532375/5259610

`T` separator seems as better option to me.